scripts/skills/effects/rf_life_leech_effect.nut

```diff
@@ -0,0 +1,100 @@
+this.rf_life_leech_effect <- ::inherit("scripts/skills/skill", {
+	m = {
+		LifeLeechValue = 1.0,   // 1.0 equals to 100% damage leeched as life
+        LeechDistance = 1,
+		ForceHeal = false   // Temporary helper variable
+	},
+	function create()
+	{
+		this.m.ID = "effects.rf_life_leech";
+		this.m.Name = "Life Leech";
+		this.m.Description = "This character recovers Hitpoints when dealing damage to enemies with blood.";
+		this.m.Icon = "skills/status_effect_09.png";
+		this.m.IconMini = "";   // We currently have no fitting mini-icon for this
```

Comment:
I can create one, using the bigger icon

Reply:
But I also argue that this does not need a mini-icon.
Mini-Icons should be used sparingly to not overload the UI.

Especially effects which are always active like Life-Leech should therefor be the first to **not** be using mini-icons

Reply:
Sure, yes that is fine.